pkg/oci/utils_test.go

```diff
@@ -86,18 +86,24 @@ func TestMinimalPodConfig(t *testing.T) {
 			Destination: "/proc",
 			Type:        "proc",
 			Options:     nil,
+			IgnoreMount: false,
+			HostPath:    "",
 		},
 		{
 			Source:      "tmpfs",
 			Destination: "/dev",
 			Type:        "tmpfs",
 			Options:     []string{"nosuid", "strictatime", "mode=755", "size=65536k"},
+			IgnoreMount: false,
+			HostPath:    "",
 		},
 		{
 			Source:      "devpts",
 			Destination: "/dev/pts",
 			Type:        "devpts",
 			Options:     []string{"nosuid", "noexec", "newinstance", "ptmxmode=0666", "mode=0620", "gid=5"},
+			IgnoreMount: false,
+			HostPath:    "",
 		},
 	}
 
```

Comment:
Yes, I was planning to send this is a separate structure say "mounts" to hyperstart. But I suppose i could get away with extending the current fsmap type to handle this as well. I'll add the system mounts handling in a separate PR.